domain/src/main/java/org/folio/spring/domain/controller/JsonSchemasController.java

```diff
@@ -31,14 +35,23 @@ public JsonSchemasController(JsonSchemasService jsonSchemasService) {
   public ResponseEntity<Object> getSchemas(
   // @formatter:off
     @RequestParam(value = "path", required = false) Optional<String> path,
-    @RequestHeader(value = "x-okapi-url", required = true) String okapiUrl
+    @RequestHeader(value = "x-okapi-url", required = true) String okapiUrl,
+    @RequestHeader(value = "accept", required = false) String accept
```

Comment:
Are there `@GetMapping` or `@RequestMapping` annotation parameters that can provide the same effect here?

Reply:
There is a `provides` option for `@GetMapping`, but it operates differently and with a different order of operation. This is not ideal and is less friendly.
The `@GetMapping` `provides` also fails to handle the conditional logic.
In theory, this could be handled by created a new function for each conditional logic, but only if they overlap.

However, it does looks like I should be returning a `406` rather than a `415`.
I'll update this PR shortly with the change from `415` to `406` , aka: `Not Acceptable`.

Reply:
I would prefer an abstraction to handle repetitive conditioning, especially in controller endpoints. Perhaps a custom argument resolver, or custom before and after point cut.